tidb-cloud/create-tidb-cluster.md

```diff
@@ -23,7 +23,7 @@ TiDB Cloud provides the following two cluster tier options. Before creating a Ti
 
 - Developer Tier
 
-    The TiDB Cloud Developer Tier is a one-year free trial of [TiDB Cloud](https://pingcap.com/products/tidbcloud), the fully managed service of TiDB. You can use Developer Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for non-commercial datasets.
+    The TiDB Cloud Developer Tier is a one-year free trial of [TiDB Cloud](https://en.pingcap.com/tidb-cloud/), the fully managed service of TiDB. You can use Developer Tier clusters for non-production workloads such as prototype applications, hackathons, academic courses, or to provide a temporary data service for non-commercial datasets.
```

Comment:
Is term "non-commercial datasets" here as sort of disclaimer? It makes me a little bit scared that you might sue me for commercial usage.

Reply:
Yes, it might cause that confusion. "for non-commercial datasets" could be updated to "for your datasets".